packages/mirrorful/editor/src/pages/index.tsx

```diff
@@ -14,7 +32,13 @@ export default function Editor() {
         />
         <link rel="icon" href="/favicon.ico" />
       </Head>
-      <Dashboard
+      <Layout isLoading={isLoading}>
+        <ColorPaletteSection
+          colors={colors}
+          onUpdateColors={handleUpdateColors}
+        ></ColorPaletteSection>
```

Comment:
super nit, but I'm surprised the linter doesn't force us to make this just one tag, i.e.,

```javascript
        <ColorPaletteSection
          colors={colors}
          onUpdateColors={handleUpdateColors}
        />
```